Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -0,0 +1,265 @@
+    private let configuration: ARWorldTrackingConfiguration
+    private var initialTransformationIsSet: Bool { initialTransformation != nil }
+    
+    /// Creates a table top scene view.
+    /// - Parameters:
+    ///   - anchorPoint: The location point of the ArcGIS Scene that is anchored on a physical surface.
+    ///   - translationFactor: The translation factor that defines how much the scene view translates
+    ///   as the device moves.
+    ///   - clippingDistance: Determines the clipping distance in meters around the camera. A value
+    ///   of `nil` means that no data will be clipped.
+    ///   - sceneView: A closure that builds the scene view to be overlayed on top of the
+    ///   augmented reality video feed.
+    /// - Remark: The provided scene view will have certain properties overridden in order to
+    /// be effectively viewed in augmented reality. Properties such as the camera controller,
+    /// and view drawing mode.
+    public init(
+        anchorPoint: Point,
+        translationFactor: Double,
+        clippingDistance: Double?,
+        @ViewBuilder sceneView: @escaping (SceneViewProxy) -> SceneView
+    ) {
+        self.sceneViewBuilder = sceneView
+        
+        let initialCamera = Camera(location: anchorPoint, heading: 0, pitch: 90, roll: 0)
+        let cameraController = TransformationMatrixCameraController(originCamera: initialCamera)
+        cameraController.translationFactor = translationFactor
+        cameraController.clippingDistance = clippingDistance
+        _cameraController = .init(initialValue: cameraController)
+        
+        configuration = ARWorldTrackingConfiguration()
+        configuration.worldAlignment = .gravityAndHeading
+        configuration.planeDetection = [.horizontal]
+    }
+    
+    public var body: some View {
+        ZStack {
+            ARSwiftUIView(proxy: arViewProxy)
+                .onDidUpdateFrame { _, frame in
+                    guard let sceneViewProxy else { return }
+                    updateLastGoodDeviceOrientation()
+                    sceneViewProxy.updateCamera(
+                        frame: frame,
+                        cameraController: cameraController,
+                        orientation: lastGoodDeviceOrientation,
+                        initialTransformation: initialTransformation ?? .identity
+                    )
+                    sceneViewProxy.setFieldOfView(
+                        for: frame,
+                        orientation: lastGoodDeviceOrientation
+                    )
+                }
+                .onAddNode { _, node, anchor in
+                    addPlane(with: node, for: anchor)
+                }
+                .onUpdateNode { _, node, anchor in
+                    updatePlane(with: node, for: anchor)
+                }
+                .onSingleTapGesture { screenPoint in
+                    guard let sceneViewProxy,
+                          !initialTransformationIsSet else { return }
```

Comment:
This is purely stylistic, so take it or leave it. When the guard statement is multiple lines I prefer the else on a separate line.  Same suggestion applies to other multi-line guard statements below.

```suggestion
                    guard let sceneViewProxy,
                          !initialTransformationIsSet
                    else { return }
```